ietf-layer0-types-ext.yang

```diff
@@ -316,6 +316,24 @@ module ietf-layer0-types-ext {
       "OTUCn (n x 105.25 Gb/s)";
   }
 
+  identity type-power-mode {
+    description
+      "power equalization mode used within the 
+       OMS and its elements";
+  }
+
+  identity power-spectral-density {
+    base type-power-mode;
+    description
+      "all elements must use power spectral density (W/Hz)";
+  }
+
+  identity channel-power {

```

Comment:
shouldn't this be carrier instead of channel ?

Reply:
Addressed in https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext/pull/25/commits/131b8f8939b6aa5cd6ed79608ff5263589ef687a